pages/index.vue

```diff
@@ -145,14 +145,12 @@ export default {
         I18nPageWrapper,
         Intro,
     },
-    async asyncData({ store, payload }) {
-        if (payload) return { sponsorsData: payload }
-        await store.dispatch('$getSponsorsData')
-        const sponsorsData = store.state.sponsorsData
-        return {
-            sponsorsData,
+    async asyncData({ store }) {
+        if (process.client) {
+            await store.dispatch('$getSponsorsData')
         }
     },
+    fetchOnServer: false,
```

Comment:
Due to asyncData is used for generating static html in previous years,
i think asyncData should not be modified

Reply:
If I remember correctly, we used to fetch data when `created`.

So I just changed it to the previous workable version, and keep `asyncData` as well if we need to archive the site

Reply:
See the changes in `page/index`

https://github.com/pycontw/pycontw-frontend/commit/0293e46cd7ac514f4664666b8e0c6cf2278e5017